web/blog/2025-06-20-oauth-security-incident.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Incident Report: Case insensitive OAuth IDs  vulnerability in Wasp'
+authors: [miho]
+image: /img/security-incident/security-incident.png
+tags: [security, oauth, keycloak, wasp]
+---
+
+## Introduction
+
+We received a report on a security vulnerability in Wasp auth related to our OAuth support in Wasp `0.16.5` and earlier.
```

Comment:
I like how we did in the last one:

> On Feb 12th, 2025, we learned about a security vulnerability in the updateCurrentUser function of our Open SaaS template. Users of apps built with Open SaaS can exploit this vulnerability to edit any field in their own User database record, including fields they weren’t supposed to have write permissions for, like subscriptionPlan and isAdmin.

Maybe rewrite it a bit with that as inspiration. I think the very start is a nice format: "On <date>, we learned about ...".